src/main/twirl/com/lightbend/lagom/docs/getstartedscala.scala.html

```diff
@@ -47,7 +47,7 @@
 sbt runAll</code></pre>
 
                     <p>
-                        See <a href="@context.path/documentation/@context.currentDocsVersion/scala/GettingStarted.html#Run-Hello-World">the documentation</a>
+                        See <a href="@context.path/documentation/@context.currentDocsVersion/scala/IntroGettingStarted.html#Run-Hello-World">the documentation</a>
```

Comment:
It seems that it should have been `IntroGetStarted.html`, not `IntroGettingStarted.html`